strings/src/main/java/com/revenuecat/purchases/strings/AttributionStrings.kt

```diff
@@ -1,5 +1,36 @@
 package com.revenuecat.purchases.strings
 
 object AttributionStrings {
-    const val NO_SUBSCRIBER_ATTRIBUTES_TO_SYNCHRONIZE = "No subscriber attributes to synchronize."
+    const val ATTRIBUTE_ERROR = Emojis.SAD_CAT_EYES + Emojis.ERROR + " There were some subscriber attributes errors: %s"
+    const val ATTRIBUTES_SYNC_SUCCESS = Emojis.HEART_CAT_EYES + " Subscriber attributes synced successfully for " +
```

Comment:
is this explicit emoji usage going to be replaced with the convenience methods in https://github.com/RevenueCat/purchases-android/pull/219/?